docker/puppetdb/docker-entrypoint.sh

```diff
@@ -4,7 +4,7 @@ set -e
 
 for f in /docker-entrypoint.d/*.sh; do
     echo "Running $f"
-    chmod +x "$f"
+    chmod +x "$f"; sync
     "$f"
 done
```

Comment:
I'd be a little tempted to break this up so that we only need one sync, i.e.:
```
chmod +x /docker-entrypoint.d/*.sh
# sync prevents aufs from sometimes returning EBUSY if you exec right after a chmod
sync
for f in /docker-entrypoint.d/*.sh; do
    ...
done
```

Reply:
Makes sense!